frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/emoji_picker/emoji_shortcut_event.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+import '../../layout/sizes.dart';
+import 'emoji_menu_item.dart';
+
+CommandShortcutEvent showEmojiPickerEvent(BuildContext context) =>
+    CommandShortcutEvent(
+      key: 'Show emoji picker',
+      command: 'ctrl+alt+e',
+      macOSCommand: 'cmd+alt+e',
+      handler: (state) =>
+          _showEmojiSelectionMenuShortcut(Overlay.of(context), state, context),
+    );
+
+KeyEventResult _showEmojiSelectionMenuShortcut(
+  OverlayState container,
+  EditorState editorState,
+  BuildContext context,
+) {
+  final selectionService = editorState.service.selectionService;
+  final selectionRects = selectionService.selectionRects;
+  final editorOffset =
+      editorState.renderBox?.localToGlobal(Offset.zero) ?? Offset.zero;
+  final editorHeight = editorState.renderBox!.size.height;
+
+  // show below default
+  var showBelow = true;
+  var alignment = Alignment.bottomLeft;
```

Comment:
you can get the context from the selected node, then don't need to pass the context. Also, does the emoji picker only appear when the selection is collapsed?

Reply:
Yes, it works only when the menu is collapsed. Should it work with the open menu?